metabase/metabase
# Improve password reset flow

Epic: https://github.com/metabase/metabase/issues/19476

- Changes visuals
- Adds password validation
- Converts to typescript and styled components

<img width="580" alt="Screenshot 2021-12-27 at 19 11 22" src="https://user-images.githubusercontent.com/8542534/147502172-282f486d-8ddb-44ab-83e0-e6a6f4a5e9c6.png">
<img width="550" alt="Screenshot 2021-12-27 at 19 12 01" src="https://user-images.githubusercontent.com/8542534/147502176-836a1d45-03dc-4933-bf98-e9185e6e7ff4.png">

How to test:
- Configure email in Metabase. You can use `maildev` for this https://github.com/metabase/metabase/blob/f94c3c96f69bcdd0b1f297553f94d02210b17218/frontend/test/__support__/e2e/helpers/e2e-email-helpers.js#L2
- Open `/auth/forgot_password` and submit the form with your email
- Click the link in the email and reset the password
- Make sure it was changed correctly
